packages/client/utils/AtlassianManager.ts

```diff
@@ -571,11 +574,20 @@ export default abstract class AtlassianManager {
     return cloudNameLookup
   }
 
-  async getIssue(cloudId: string, issueKey: string, extraFieldIds: string[] = []) {
+  async getIssue(
+    cloudId: string,
+    issueKey: string,
+    extraFieldIds: string[] = [],
+    extraExpand: string[] = []
+  ) {
     const baseFields = ['summary', 'description']
-    const reqFields = [...baseFields, ...extraFieldIds].join(',')
+    const reqFields = extraFieldIds.includes('*all')
+      ? '*all'
+      : [...baseFields, ...extraFieldIds].join(',')
+    const baseExpand = ['renderedFields']
+    const expand = [...baseExpand, ...extraExpand].join(',')
```

Comment:
might be personal opinion, but i find it easier to understand if 'renderedFields' was hardcoded here instead of making the extra variable.